torch_xla/experimental/distributed_checkpoint.py

```diff
@@ -88,12 +83,14 @@ def set_up_planner(self, state_dict: STATE_DICT_TYPE,
     # types that can be handled by the default planner, and ensure all sharded
     # tensors are wrapped in XLAShardedTensor
     state_dict, self.mappings = flatten_state_dict(state_dict)
-    state_dict = tree_map(xs.wrap_if_sharded, state_dict)
+    state_dict = _sharded_cpu_state_dict(state_dict)
```

Comment:
We don't want to call _sharded_cpu_state_dict within the planner, this should still just tree_map `wrap_if_sharded`

Reply:
this is what I had earlier but the tests failed so I changed it